src/functions/node.php

```diff
@@ -21,3 +24,13 @@ function node_view($node, $view_mode = 'full', $langcode = null)
       ->getViewBuilder('node')
       ->view($node, $view_mode, $langcode);
 }
+
+function node_types_rebuild(): void
+{
+    $tags = \Drupal::config('node_type')->getCacheTags();
+    $tags[] = 'config:node_type_list';
+    Cache::invalidateTags($tags);
+    $storage = \Drupal::service('config.storage');
+    assert($storage instanceof StorageCacheInterface);
+    $storage->resetListCache();
```

Comment:
Instead of this, we could load all `node_type` entities, collect their `getCacheTags` and use `Cache::mergeTags` to invalidate all of their individual cache tags.